great_expectations/core/expectation_suite.py

```diff
@@ -754,8 +754,7 @@ def _validate_expectation_configuration_before_adding(
     ):
         try:
             class_ = get_expectation_impl(expectation_configuration.expectation_type)
-            expectation = class_()
-            expectation.validate_configuration(expectation_configuration)
+            _ = class_(expectation_configuration)  # Implicitly validates in constructor
```

Comment:
A little unrelated but can you fix the docstring for `ExpectationConfiguration`? I think `denies` should be `defines`. [See here](https://github.com/great-expectations/great_expectations/blob/d7ff25553aac1a05736c4ca2979f63d5b17745e7/great_expectations/core/expectation_configuration.py#L135).

Reply:
nit: you can remove the `_ =`. I think this will allocate a variable on the stack that you don't need. Your comment explains why are you doing this.